scss/_variables-dark.scss

```diff
@@ -53,7 +53,7 @@ $headings-color-dark:               inherit !default;
 $link-color-dark:                   tint-color($primary, 40%) !default;
 $link-hover-color-dark:             shift-color($link-color-dark, -$link-shade-percentage) !default;
 $code-color-dark:                   tint-color($code-color, 40%) !default;
-
+$btn-link-focus-shadow-rgb-dark:    to-rgb(mix(color-contrast($link-color-dark), $link-color-dark, 15%)) !default;
```

Comment:
I feel like the focus shadow is less visible than the one on main. Maybe as a default value we can use the light version of the shadow ?
Furthermore, it seems like we don't lighten the shadow even on dark mode so I'd definitely keep the light one.